src/kohlrahbi/conditions/command.py

```diff
@@ -14,16 +14,46 @@
 To use this command-line interface, run the script and provide the necessary options as command-line arguments.
 
 Example:
-$ python command.py --edi-energy-mirror-path /path/to/edi_energy_mirror/repo
- --output-path /path/to/output --format-version FV2310 --assume-yes
+$ python command.py --input-path /path/to/input --output-path /path/to/output --format-version FV2310 --assume-yes
 """
 
+# pylint: disable=duplicate-code
```

Comment:
this is not a good sign ;)

Reply:
I agree. However, this stems from the fact that the cli structure for `ahb`s and `conditions` are similar/identical. I added a comment that refers to this (6b989dfff68b25d817ea1943e18e923604f557db).